deploy/helm/smg/templates/hpa-router.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "smg.fullname" . }}-router
+  labels:
+    {{- include "smg.labels" . | nindent 4 }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ include "smg.fullname" . }}-router
+  minReplicas: {{ .Values.router.autoscaling.minReplicas }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)